apps/web-app/README.md

```diff
@@ -1,5 +1,17 @@
 # Challenge Registry Web App
 
+## Development
+
+Start the MongoDB instance that hosts the data managed by the challenge registry
+API. The following command must be run from the root of the workspace.
+
+    docker compose up api-db -d
```

Comment:
After getting updates from upstream, I have to use `docker compose up challenge-registry-api -d` to start api now.